Microsoft.HealthVault.Fhir.UnitTests/ToHealthVaultTests/ObservationToHealthVaultWeightTests.cs

```diff
@@ -36,6 +36,21 @@ public void WeightToHealthVault_Successful()
         }
 
         [TestMethod]
+        public void HeightToHealthVault_Successful()
```

Comment:
I was planning to open a new PR to change the filename/class to "ObservationToHealthVaultTests" since it's a code cleanup item. I can do it in this PR if you prefer.